### ISSUE TITLE
How many images and how many epochs are required to fine tune LORA for stable diffusion on custom image dataset

### ISSUE THREAD

**Issue by atharmzaalo2023** [huggingface/diffusers#3881]
I am trying to finetune LORA on a movie dataset , but I am using custom dataset which has 3-4 movie characters , instead of using the actual names of the actor we are using in movie name of the characters , how big the dataset would be required in terms of total number of images, and number of images per character and how many epochs would be required to fine tune this LORA model .
PS: I have already tried fine tuning with 200 images of a single character for 100,250 and 500 Epochs but the results are very bad , can anyone please provide some suggestion @patrickvonplaten  @sayakpaul

**Comment by patrickvonplaten** [comment]
cc @sayakpaul

**Comment by sayakpaul** [comment]
Hi @atharmzaalo2023! Thanks for reporting this.

Does the dataset have characters concerning faces? If so, I welcome you to check out our blog post: https://huggingface.co/blog/dreambooth. It has some really good pointers. 

Also cc: @pcuenca @apolinario.

**Comment by atharmzaalo2023** [comment]
Hi @sayakpaul ,
Thanks for pointing out to the blog , I will definitely check it out , Our dataset does have lot of images of characters concerning faces , we have for starters taken only single characters and have approximately 200 images with detailed prompts also some images are of full characters but most of them are closeup shot and does indeed have face of character visible

**Comment by sayakpaul** [comment]
Thanks for letting us know. 

DreamBooth might struggle with multi-concept training, however. So, I suggest the following:

* [Custom Diffusion](https://huggingface.co/docs/diffusers/main/en/training/custom_diffusion)
* [SVDiff](https://github.com/mkshing/svdiff-pytorch) (cc: @mkshing)

**Comment by atharmzaalo2023** [comment]
Hey @sayakpaul  I read the blog that you have mentioned earlier about dreambooth , I would try custom diffusion and svdiff also however I would still like to give dreambooth a try . You have mentioned in that blog you tried to train dreambooth with Kramer character from Seinfeld images , the faces images specifically , You have mentioned that typically learning rate of 2e-6 with train_steps~1200 give best results , also if you can please tell what was the size of the dataset of Kramer character , can you provide me how many samples do you use and whether the images were of the same scene or taken from different scene . If you could provide that I could give dreambooth a try

**Comment by sayakpaul** [comment]
The sample size was ~10.
